common/v2/features/Layout/Layout.tsx

```diff
@@ -42,7 +43,7 @@ const STop = styled.div`
 `;
 
 const SContainer = styled('div')`
-  padding: ${SPACING.BASE};
+  ${({ noPadding }) => (noPadding ? `padding: ${SPACING.BASE} 0;` : `padding: ${SPACING.BASE};`)};
```

Comment:
Maybe it's easier to read with `padding: ${ ({noPadding}) => noPadding ? `${SPACING.BASE} 0` :  ${SPACING.BASE}};` ?